packages/twenty-front/src/modules/settings/data-model/objects/forms/components/SettingsDataModelObjectAboutForm.tsx

```diff
@@ -316,14 +323,13 @@ export const SettingsDataModelObjectAboutForm = ({
                       title={t`Synchronize Objects Labels and API Names`}
                       description={t`Should changing an object's label also change the API?`}
                       checked={value ?? true}
-                      disabled={
-                        isDefined(objectMetadataItem) &&
-                        !objectMetadataItem.isCustom
-                      }
                       advancedMode
                       onChange={(value) => {
                         onChange(value);
-                        if (value === true) {
+                        if (
+                          value === true &&
+                          (!objectMetadataItem || objectMetadataItem?.isCustom)
```

Comment:
If I set the value to true for a standard object, then the Label should be updated back to its value as you rightly pointed out on Discord. Is that done purely server-side and then it's updated with the response? I'm reviewing the code but haven't tested it yet

Reply:
It should be purely server-side, since translations of labels are server-side. Label inputs should updated with the response (currently the update are propagated to the rest of the app, but not the label inputs, as I mentioned in the PR description)